pkg/suite/suite.go

```diff
@@ -81,3 +94,82 @@ func (s Suite) FindTests(pattern string) ([]runtime.TestCase, error) {
 func (s Suite) GetGlobalConfig() runtime.GlobalTestConfig {
 	return s.Config
 }
+
+// MergeConfigs overlays a global configuration over an entire suite
```

Comment:
overwrites is more clear as overlay would imho indicate that a layer is saved somewhere.